tooling/build/scripts/publishing/index.ts

```diff
@@ -289,24 +292,21 @@ async function writeContentToFile(
   parentId: number | null,
 ) {
   try {
-    if (!fullPermalink) {
-      console.error("Error: fullPermalink is undefined or empty for resource")
-      return
-    }
-
     // NOTE: do a join with ./ here so that
     // we don't end up with an absolute path to a special unix folder
-    const sanitizedPermalink = path.join(
-      "./",
-      path
-        // NOTE: normalization here will remove dual backslashes
-        // and also strip .. filepaths except as a prefix
-        .normalize(fullPermalink)
-        // NOTE: this matches on a leading ../
-        // or a leading ..\
-        // or a plain .. without any paths
-        .replace(/^(\.\.(\/|\\|$))+/, ""),
-    )
+    const sanitizedPermalink = !fullPermalink
+      ? "index"
```

Comment:
double checking - iosn't this supposed to be `_index`?

Reply:
Ah right yes, updated in 33df722fd3307f6cad2229dc88269df08f82a63d.